test/jest_integration/runIdpServer.js

```diff
@@ -17,15 +17,21 @@ const { runServer } = require('saml-idp');
 
 const { generate } = require('selfsigned');
 
+const minimist = require('minimist');
```

Comment:
Don't you need to add this to package.json and yarn.lock? I am trying to understand why the build/install isn't complaining unless this is getting pulled in from somewhere else as well, but maybe we want to explicitly specify it?

Reply:
Update: I found it in the yarn.lock file, so that's while build/install isn't complaining. I guess it is a sub-dependency of an existing dependency, but I think we should add it explicitly to protect ourselves from future deprecations/updates, and lock it to a specific major and or minor.

Reply:
Added minimist as a dev dependency